tox.ini

```
@@ -1,8 +1,15 @@
 [tox]
+<<<<<<< HEAD
 envlist = {nautilus,rhcs}-{centos,ubuntu}-{container,non_container}-{all_daemons,collocation,update,bluestore_lvm_osds,lvm_osds,shrink_mon,shrink_osd,lvm_batch,add_osds,add_mdss,rgw_multisite,purge,storage_inventory,lvm_auto_discovery}
   {nautilus,rhcs}-{centos,ubuntu}-container-{ooo_collocation,podman}
   {nautilus,rhcs}-{centos,ubuntu}-non_container-{switch_to_containers}
   nautilus-rhel-container-podman
+=======
+envlist = {dev,rhcs}-{centos,ubuntu}-{container,non_container}-{all_daemons,collocation,bluestore_lvm_osds,lvm_osds,shrink_mon,shrink_osd,lvm_batch,add_mons,add_osds,add_mdss,rgw_multisite,purge,storage_inventory,lvm_auto_discovery}
```

```suggestion
envlist = {nautilus,rhcs}-{centos,ubuntu}-{container,non_container}-{all_daemons,collocation,bluestore_lvm_osds,lvm_osds,shrink_mon,shrink_osd,lvm_batch,add_mons,add_osds,add_mdss,rgw_multisite,purge,storage_inventory,lvm_auto_discovery}
```
It's fine to remove `update` from that list since it's present in `tox-update.ini`